### PR TITLE
sync ps/assert output

### PR DESCRIPTION
**Pull request by flyingfish89**
## Summary
sync ps/assert output

### CODE REVIEW

File: sched/misc/assert.c [hunk 1]
```diff
@@ -221,6 +234,9 @@ static void show_stacks(FAR struct tcb_s *rtcb)
 static void dump_task(FAR struct tcb_s *tcb, FAR void *arg)
 {
   char args[64] = "";
+  char state[32];
+  char str[64];
```
Comment (xiaoxiang781216) [inline sched/misc/assert.c:238]: why use the different size for str and state?

File: sched/misc/assert.c [hunk 2]
```diff
@@ -255,13 +271,29 @@ static void dump_task(FAR struct tcb_s *tcb, FAR void *arg)
 
   group_argvstr(tcb, args, sizeof(args));
 
+  /* get the task_state */
+
+  nxsched_get_stateinfo(tcb, state, sizeof(state));
+  if ((s = strchr(state, ',')) != NULL)
+    {
+      *s = '\0';
+      snprintf(str, sizeof(str), "%s  %s", state, s + 1);
```
Comment (xiaoxiang781216) [inline sched/misc/assert.c:280]: why use the two spaces

File: sched/misc/assert.c [hunk 3]
```diff
@@ -255,13 +271,29 @@ static void dump_task(FAR struct tcb_s *tcb, FAR void *arg)
...
+    }
+  else
+    {
+      strlcpy(str, state, sizeof(str));
+    }
+
   /* Dump interesting properties of this task */
 
-  _alert("  %4d   %4d"
+    _alert("   %4d %5d"
```
Comment (xiaoxiang781216) [inline sched/misc/assert.c:289]: why add two spaces

File: sched/sched/sched_get_stateinfo.c [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+void nxsched_get_stateinfo(FAR struct tcb_s *tcb, FAR char *state,
+                           size_t length)
+{
+  /* if the state is waiting mutex */
+
+  if (tcb->task_state == TSTATE_WAIT_SEM &&
+      ((FAR sem_t *)(tcb->waitobj))->flags & SEM_TYPE_MUTEX)
+    {
+      snprintf(state, length, "Waiting,Mutexholder:%d",
```
Comment (xiaoxiang781216) [inline sched/sched/sched_get_stateinfo.c:88]:
```suggestion
      snprintf(state, length, "Waiting,Mutex:%d",
```

File: sched/misc/assert.c (hunk 2 above)
Comment (flyingfish89) [reply sched/misc/assert.c:280]: because i need change  "waiting,mutex:9" -> "waiting  mutex:9" to Keep it consistent with ps

Comment (xiaoxiang781216) [reply sched/misc/assert.c:280]: one space is enough?

Comment (xiaoxiang781216) [reply sched/misc/assert.c]: Yes.

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**